src/main/frontend/db/model.cljs

```diff
@@ -1235,9 +1235,16 @@
              (sort-by-left-recursive)
              db-utils/group-by-page)))))
 
+(defn- contains-cjk?
+ "Checks if text contains character in unicode range for CJK and returns a boolean"
+ [text]
+  (boolean (re-find (re-pattern "[\\u3040-\\u30ff\\u3400-\\u4dbf\\u4e00-\\u9fff\\uf900-\\ufaff\\uff66-\\uff9f]") text)))
+
 (defn- pattern [name]
   (re-pattern (str "(?i)(^|[^\\[#0-9a-zA-Z]|((^|[^\\[])\\[))"
+                   "^"
                    (util/regex-escape name)
+                   (if (contains-cjk? (util/regex-escape name)) "\\d*" nil)
```

Comment:
```suggestion
                   (if (contains-cjk? name) "\\d*" nil)
```

Reply:
after further inspection. The name does need to be escaped in the case the notebook might have any special characters.